lib/common/api/deprecate.ts

```diff
@@ -43,7 +43,7 @@ const deprecate: ElectronInternal.DeprecationUtil = {
     return function (this: any) {
       warn();
       fn.apply(this, arguments);
-    };
+    } as any;
```

Comment:
does this work? same effect i suppose from the outside, but it's a bit better documentation
```suggestion
    } as any as typeof fn;
```